Frontend/src/components/rankings/DailyRanking.vue

```diff
@@ -9,17 +9,17 @@
               v-for="(flight, index) in flights.slice(0, maxRows)"
               v-bind:item="flight"
               v-bind:index="index"
-              v-bind:key="flight._id"
-              @click="routeToFlight(flight.flightId)"
-              @mouseover="updateHighlightedFlight(flight.flightId)"
+              v-bind:key="flight.id"
+              @click="routeToFlight(flight.id)"
+              @mouseover="updateHighlightedFlight(flight.id)"
               @mouseleave="updateHighlightedFlight(null)"
             >
               <td scope="row">{{ index + 1 }}</td>
-              <td>{{ flight.pilot }}</td>
-              <td>{{ flight.takeoff }}</td>
-              <td>{{ flight.distance }} km</td>
-              <td>{{ flight.taskType }}</td>
-              <td>{{ flight.points }} P</td>
+              <td>{{ flight.User.name }}</td>
+              <td>{{ flight.takeoff.name }}</td>
+              <td>{{ flight.flightDistance }} km</td>
+              <td>{{ flight.flightType }}</td>
```

Comment:
Type hätte ich in ein Icon übersetzt

Reply:
Genau, das wird noch kommen.

Reply:
Muss noch gute Icons dafür finden

Reply:
Einfach mal auf die TODO Liste setzen ;-)